### ISSUE TITLE
[Android 11] Phonebook resets after phone reloading.

### ISSUE THREAD

**Issue by CortexReaver** [SimpleMobileTools/Simple-Dialer#676]
Once you added someone to a phonebook, If you decided to reload your phone your newly added contacts will be gone after it. Also it seems like your app doesn't count newly added contacts as "saved on phone" (it just shows me 0), neither can I save on a SIM card.

**Comment by tibbi** [comment]
What app and OS version are you using? Are you saving those contacts on the Phone, SIM or somewhere else?

**Comment by CortexReaver** [comment]
This one: https://f-droid.org/en/packages/com.simplemobiletools.dialer/ Simple Dialer, the same as the name of this github project. OS version is in the title: it's Android 11.
I've realized just now that you can choose where to save it right in the end of a contact creation menu :) So, saving on phone seems to be working fine, but not on the SIM-card.

**Comment by tibbi** [comment]
Simple Dialer cannot manage or create contacts, you must be using some other Contacts app for it.

**Comment by CortexReaver** [comment]
Yes, I've already got "Simple contacts" for this purpose. Sorry, all this android stuff is relatively new to me. I think I've got disabled all the default android contacts\dialers\whatever and go straight to fdroid to get the FOSS stuff. So right now I've got pretty much "the full pack" of your apps: Simple Dialer, Simple contacts, Gallery and clock.
So, do I have to create a separate issue in https://github.com/SimpleMobileTools/Simple-Contacts ? Might it be the "Optimize battery" option that is causing the bug I described?
